test/microbench/org/apache/cassandra/test/microbench/CompactionBench.java

```diff
@@ -35,7 +35,7 @@
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @Warmup(iterations = 25, time = 1, timeUnit = TimeUnit.SECONDS)
-@Measurement(iterations = 5, time = 2, timeUnit = TimeUnit.SECONDS)
+@Measurement(iterations = 5, time = 50, timeUnit = TimeUnit.SECONDS)
```

Comment:
Looks like some of your local testing that went into the commit. Please remove that change, because it is unrelated.